stories/Welcome.mdx

```diff
@@ -1,7 +1,138 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
 
-<h1>Welcome</h1>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-xl mx-auto text-pretty flex flex-col space-y-36 pb-36">
+    <header class="header flex justify-center align-middle">
+      <a href="https://www.gridironsurvivor.com" target="_blank" rel="noreferrer" class="w-2/6 h-fit">
+        <Image class="w-full" src={gridironLogo} alt='Gridiron Survivor Football Logo' />
+      </a>
+    </header>
+    <section class="hero space-y-12">
+      <h1 class="pl-16">What is Gridiron Survivor?</h1>
+      <p class="text-xl text-pretty max-w-screen-lg mx-auto">
+        Gridiron Survivor is the ultimate pick'em league app. Track your picks, see how you stack up against your friends, and hope to be crowned champion! Developed by a talented team of junior developers using technologies such as Next.js, React, and Tailwind CSS, Gridiron Survivor is the perfect tool for any football fan looking to take their pick'em game to the next level.
+        <br />
+        Gridiron Survivor is currently in its first Alpha for the 2024-2025 NFL Season! 1st, 2nd, and 3rd place winners of the season get spoiled with prizes from our generous sponsors.
+      </p>
+    </section>
+    <section class="sponsors-section h-fit">
+      <h2 class="pl-16 pb-8">Sponsors</h2>
+      <div class="sponsors-card-container flex flex-row flex-wrap justify-between max-w-screen-lg mx-auto">
+        <a href="https://appwrite.io/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={appwriteLogo} />
+          <h3>Appwrite</h3>
+        </a>
+        <a href="https://blogrecorder.com/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={blogRecorderLogo} />
+          <h3>Blog Recorder</h3>
+        </a>
+        <a href="https://www.frontendmentor.io/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={frontendMentorLogo} />
+          <h3>Frontend Mentor</h3>
+        </a>
+        <a href="https://www.gitkraken.com/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={gitkrakenLogo} />
+          <h3>GitKraken</h3>
+        </a>
+        <a href="https://usepastel.com/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={pastelLogo} />
+          <h3>Pastel</h3>
+        </a>
```

Comment:
Feels like we could automate this a bit with a map array, where we have a list of objects with the sponsor name, sponsor link, and sponsor logo and loop through those.  This would also make it easier to add future sponsors.

Reply:
100% agreed.

I did try to create something similar for the team member section, but was unsuccessful. I wish I wrote down what the exact error was, but @ryandotfurrer had helped me with it, so I doubt the problem was with how it was built.

That being said, I will take a look into map arrays tonight and see if I can get that built.

I ran into a handful of limitations while working on this. Not sure if it's a limitation of Storybook, the fact that it's a .mdx file, or a skill issue.

Reply:
Looking at [MDX docs])https://mdxjs.com/docs/using-mdx/#how-mdx-works), [storybook array of objects](https://stackoverflow.com/questions/64986941/storybook-6-how-to-set-array-of-objects), you might have to import it from a JSX or TSX file. I would be open to workshoping this with you sometime though.

Reply:
This is complete with my latest push.